chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
This isn't really a question of inconsistency. `let` should definitely be used for block scope, as we've done for years. But that doesn't mean that it's inconsistent to use `var for function scope if we agree that there are benefits.